### PR TITLE
Holy mother of spaghetti

### LINKED ISSUE #68: Remove temp code from Inventory when ready

### LINKED ISSUE #79: Market starting values
market should start with 16 units of energy, no ore and 12 roboticons

### PR DESCRIPTION
**Pull request by Mikeywalsh**
I am so sorry
The game works
But at what cost...

closes #79 
closes #68

### CODE REVIEW

**Comment by RobinStephenson** [comment]
Also closes #3 #4 #5 #6 #7 #8 #9 #11

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Jamaican Bobsleigh Team
+# Jamaican Bobsleigh Team [![Build Status](https://travis-ci.com/RobinStephenson/jbt.svg?token=HjdBafwzwwqM31btobaT&branch=master)](https://travis-ci.com/RobinStephenson/jbt)
```
Comment (RobinStephenson) [inline README.md:1]: This link is broken so remove this.
As the project is public we have to use travis-cs.**ORG** you can copy over our travis settings onto there if you want to

**Review by RobinStephenson — APPROVED** [review]
A few issues like capitalization and the Travis badge on the readme but we can sort those out with some patches lets just get this merged

**Comment by RobinStephenson** [comment]
I made some changes to the customisations available as the requirements state that the initial customisations allow production of just their resource

**Comment by RobinStephenson** [comment]
Can we change the player names to Player 1 and Player 2

**Comment by Mikeywalsh** [comment]
Sure, I forgot about the names. Although the codes that bad that it will somehow break the game

**Comment by RobinStephenson** [comment]
@Mikeywalsh is it actually that bad?

**Comment by Mikeywalsh** [comment]
@RobinStephenson Nah it was just a joke haha

**Comment by Mikeywalsh** [comment]
I am going to have to revert the change you did though, why would ore customisation have a 1x multiplier for ore and a 0x multiplier for power, that would mean that you get 0x power(which is nothing, always) and only the standard amount of ore.

**Comment by Mikeywalsh** [comment]
To expand, I'm just going off how you implemented Tile.Produce and my understanding of the customisations. Correct me if I'm wrong but i assumed that with a base roboticon, you get the resources on the tile every turn, e.g 1x. Then with an upgrade, say power in this instance, you get the base amount of ore(1x), but double the power(2x)

**Comment by Mart500** [comment]
@Mikeywalsh The requirements state that a roboticon of the wrong type should not produce anything. Thus, for it to do so, a base production multiplier must be 0x.

**Comment by Mikeywalsh** [comment]
What do you mean by wrong type?

**Comment by Mikeywalsh** [comment]
That requirement makes zero sense btw, all tiles contain both ore and power on them, they can't have a 'wrong type'

**Comment by Mart500** [comment]
@Mikeywalsh Where was it decided that all tiles contain both ore and power? I'd assumed a tile could have possibly just ore or just power. 
Also, by wrong type, I assume to requirements talk about the type of customisation on the roboticons.  If the Roboticon has a power customisation it should not produce ore, type of thing.

**Comment by Mikeywalsh** [comment]
I took it to mean that a power customisation enhances power but doesnt hinder other resource production. Either way, we're getting into game balance and i don't think it really matters for this hand in as long as the functionality is there to implement it in the future.

**Pull request merged**